### ISSUE TITLE
slack, google groups, or gitter channel?

### ISSUE THREAD

**Issue by denfromufa** [pythonnet/pythonnet#383]
https://pythonnet.slack.com/

https://groups.google.com/forum/#!forum/pythonnet

**Comment by vmuriart** [comment]
not google groups. That looks like something still stuck in the 90s.

**Comment by vmuriart** [comment]
For reference:
https://www.quora.com/What-are-the-pros-and-cons-of-Gitter-vs-Slack
https://stackshare.io/stackups/gitter-vs-hipchat-vs-slack

Looks like `Gitter` is much more OSS friendly, and `Slack` would require to invite/request people to join.

`Slack` has built in integration to `AppVeyor`/`Travis`
`Gitter` can be hacked together to integrate with `CI`

`Slack` has a nicer UI.

I'm leaninga bit towards `Gitter`, but only because of the easier barrier of entry. Otherwise `Slack` wins.

**Comment by denfromufa** [comment]
![image](https://cloud.githubusercontent.com/assets/7870949/23009567/5883a3f4-f3dc-11e6-9ef8-f27d528a56a0.png)

**Comment by vmuriart** [comment]
So support the underdog... 🤓 
`Slack` it is then.

**Comment by vmuriart** [comment]
On a related note, we should probably try to centralize the info for the repo. ie, should consider closing or adding a notice to https://bitbucket.org/pythonnet/pythonnet/src and http://pythonnet.sourceforge.net/

Too bad http://python.net/ is taken 😢

**Comment by denfromufa** [comment]
![image](https://cloud.githubusercontent.com/assets/7870949/23009933/5d763ea6-f3de-11e6-83aa-60143328e258.png)

**Comment by denfromufa** [comment]
@vmuriart I just sent a reminder about sourceforge and copied you.

**Comment by vmuriart** [comment]
Yup, i saw the notice there. I meant we should probably add a link to the official repo. Something similar to https://bitbucket.org/hpk42/tox

**Comment by denfromufa** [comment]
pythonnet downloads on sourceforge are declining rapidly:

![image](https://cloud.githubusercontent.com/assets/7870949/23010106/4cc94200-f3df-11e6-8c32-40c9c222d1a1.png)

**Comment by stonebig** [comment]
People can now get pythonnet via "pip install pythonnet", don't they ? or via a Python distro.

**Comment by vmuriart** [comment]
yup. I'm surprised there's any activity on the `sourceforge` downloads.

**Comment by denfromufa** [comment]
@stonebig here are all download channels:

https://github.com/pythonnet/pythonnet/wiki/Installation

@vmuriart you will be surprised how much there is legacy python 2.5/2.6

**Comment by denfromufa** [comment]
I purchased this domain last year:

www.python4.net

**Comment by denfromufa** [comment]
```SQL
SELECT
  file.filename,
  COUNT(*) as total_downloads,
FROM
  TABLE_DATE_RANGE(
    [the-psf:pypi.downloads],
    DATE_ADD(CURRENT_TIMESTAMP(), -18, "day"),
    DATE_ADD(CURRENT_TIMESTAMP(), -1, "day")
  )
WHERE
  file.project = 'pythonnet'
AND
  file.filename like '%2.2.2%'
GROUP BY
  file.filename
ORDER BY
  total_downloads DESC
LIMIT 1000
```

almost 5000 downloads in 18 days:

![image](https://cloud.githubusercontent.com/assets/7870949/23010511/7a1f242a-f3e1-11e6-89b9-c513ae9e4778.png)

**Comment by vmuriart** [comment]
whoops. Sorry for derailing the topic of this issue. Looks like we are going w `Slack` then?

**Comment by denfromufa** [comment]
@vmuriart I'm fine with slack :) I'm going to invite core team. we can add slack tag on readme page later.

**Comment by vmuriart** [comment]
`shields.io` doesn't support `Slack` and I can't find a shield/badge for them. I may generate a generic badge similar to the `LICENSE` shield.

**Comment by vmuriart** [comment]
Closed by #384

**Comment by denfromufa** [comment]
public gitter channel added https://gitter.im/pythonnet/pythonnet
